package.json

```diff
@@ -21,14 +21,17 @@
   },
   "homepage": "https://github.com/northcoders/be-nc-news#readme",
   "devDependencies": {
+    "dotenv": "^16.0.3",
+    "husky": "^7.0.0",
     "jest": "^27.5.1",
     "jest-extended": "^2.0.0",
     "pg-format": "^1.0.4",
-    "husky": "^7.0.0"
+    "supertest": "^6.3.0"
   },
   "dependencies": {
```

Comment:
- I suggest that you look into the difference between dependencies and devDependencies, then move some bits around accordingly: the present setup is potentially app-breaking when it comes time to deploy your app